server.js

```diff
@@ -2,6 +2,23 @@ const express = require("express");
 const app = express();
 const cors = require("cors");
 const PORT = 4000;
+const sqlite = require("sqlite3");
+
+let db = new sqlite.Database("../shopping-list-database/database.sqlite", err => {
+    if (err) {
+        console.log(err.message)
```

Comment:
It would make sense to just terminate the service on this error. This error is critical since all operations would just fail afterwards...

You can do this with `process.exit()` (see [process.exit](https://nodejs.org/api/process.html#processexitcode)). I suggest to pass -1 as error code. In UNIX (and Linux) any non-zero exit code indicates that some error occurred. 0 means that the process (program) exited successfully (just finished working and exited).